packages/neos-ui-redux-store/src/CR/Workspaces/index.ts

```diff
@@ -75,6 +78,11 @@ const confirmDiscard = () => createAction(actionTypes.DISCARD_CONFIRMED);
  */
 const changeBaseWorkspace = (name: string) => createAction(actionTypes.CHANGE_BASE_WORKSPACE, name);
 
+/**
+ * Rebase the user workspace
+ */
+const rebaseWorkspace = (name: string) => createAction(actionTypes.REBASE_WORKSPACE, name);
```

Comment:
is there no reducer needed for REBASE_WORKSPACE? i guess because its handled via saga?